src/List.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component, Fragment } from 'react';
+
+import AssessmentCompletion from './AssessmentCompletion';
+
+class List extends Component {
+  render() {
+    return (
+      <Fragment>
+        <button onClick={this.props.history.goBack}>{"<- Back"}</button>
```

Comment:
What's the desired behaviour here? At the moment, if you add an item then you are redirected here. If you click back then you are taken to the addassessment screen. 

Probably want to have a hard coded link to take us back to the home page instead?